libs/dyn/merge/override.go

```diff
@@ -45,7 +45,7 @@ func override(basePath dyn.Path, left dyn.Value, right dyn.Value, visitor Overri
 			return dyn.InvalidValue, err
 		}
 
-		return dyn.NewValue(merged, left.Location()), nil
+		return dyn.NewValue(merged, left.Locations()), nil
```

Comment:
We should track both left and right locations here. I'll send a followup PR to do so once this is in.

Reply:
Done, added a task to our sprint board.